src/js/map/LeafletMap.js

```diff
@@ -270,26 +270,26 @@ var LeafletMap = function(params) {
     });
 
     // make the line width a fucntion of the zoom level
-    map.onLineWidth(function(){
+    map.onLineWidth(function(options){
```

Comment:
You are right about this because Im not Using the zoom function anymore to resize the links. If we eventually decide we want to resize the width also based on the zoom level, the parameter will pass the current value setup in the editor mode and it will be multiplied by the value we decide in the zoom level.